packages/manager/modules/vps/src/dashboard/migrate/migrate.service.js

```diff
@@ -0,0 +1,11 @@
+export default class VpsMigrationService {
+  /* @ngInject */
+  constructor($q, $http) {
+    this.$q = $q;
+    this.$http = $http;
+  }
+
+  migrateVps(serviceName) {
+    return this.$http.post(`/vps/${serviceName}/migration2016`);
```

Comment:
```suggestion
    return this.$http.post(`/vps/${serviceName}/migration2016`)
    .then(({data}) => data)
```
If not I think having a service is a bit overkill here if we don't do anything apart from one call